package.json

```
@@ -212,7 +212,11 @@
     "@storybook/angular": {
       "zone.js": "$zone.js"
     },
-    "replacestream": "4.0.3"
+    "replacestream": "4.0.3",
+    "@types/minimatch": "3.0.5",
+    "@electron/asar": {
+      "@types/glob": "7.1.3"
+    }
```

This should work now, both type dependencies were gone from my lockfile.
```suggestion
    "replacestream": "4.0.3"
```